### ISSUE TITLE
Rusting Schedule Availability

### ISSUE THREAD

**Issue by joeyd4329** [tobymao/18xx#10555]
Is it was possible/easy to be able to view the rusting information/column (or add this column) to the train buying step during a company's operation? 

It's nice information to have at a glance rather than having to tab over to entities to double check rusting events.

**Comment by benjaminxscott** [comment]
Totally, related to https://github.com/tobymao/18xx/issues/10276

What I'd propose is show current train limit on the `Upcoming Trains` card, and then add that card to `buy_trains` step

**Comment by joeyd4329** [comment]
Yes this would be great. Along with the train limit changes, that could be helpful too.

**Comment by benjaminxscott** [comment]
Ah yeah we probably should show the new train limit after the next train is bought, currently we don't show that under `Upcoming Trains` so it would need added (not a blocker just a change)

Currently we only show that info on the potentially huge phases chart

![image](https://github.com/tobymao/18xx/assets/1711810/5e8101d8-97f7-4d77-b704-0f45e73a4867)
